providers/celery/src/airflow/providers/celery/cli/celery_command.py

```diff
@@ -107,17 +107,7 @@ def flower(args):
 @contextmanager
 def _serve_logs(skip_serve_logs: bool = False):
     """Start serve_logs sub-process."""
-    if AIRFLOW_V_3_0_PLUS:
-        try:
-            from airflow.providers.fab.www.serve_logs import serve_logs
-        except ImportError:
-            raise ImportError(
-                "Celery requires FAB provider to be installed in order to run this command. "
-                "Please install the FAB provider by running: "
-                "pip install apache-airflow-providers-celery[fab]"
-            )
-    else:
-        from airflow.utils.serve_logs import serve_logs  # type: ignore[no-redef]
+    from airflow.utils.serve_logs import serve_logs
```

Comment:
Isn't this a breaking change for celery?
That import would not work on Airflow 2.9

Reply:
Its there: https://github.com/apache/airflow/blob/2.9.3/airflow/utils/serve_logs.py#L168-L183